boards/nucleo-wl55jc/board.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2021  Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-wl55jc
+ * @{
+ *
+ * @file        board.c
+ * @brief       Board specific implementations for the Nucleo-wl55jc board
+ *
+ *
+ * @author      Akshai M <akshai.m@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "periph/gpio.h"
+
+#define SUBGHZ_DEBUG                    0
+
+void board_init(void)
+{
+    /* initialize the CPU */
+    cpu_init();
+
+    /* Initialize the GPIO control for RF 3-port switch (SP3T) */
+    gpio_init(FE_CTRL1, GPIO_OUT);
+    gpio_init(FE_CTRL2, GPIO_OUT);
+    gpio_init(FE_CTRL3, GPIO_OUT);
```

Comment:
Is it OK to always initialize these? Would it make sense to to it only when the radio driver is present?

Reply:
this should be initialized only in the "periph" version. This could be modeled in the same way as `sx127x` with an RX_SWITCH and TX_SWITCH. We can then define these pins to GPIO_UNDEF and don't do anything in such case

Reply:
If it can be done in a generic way by the driver even better!  (this is still in `board.c` , so no harm in using board-specific pins if needed)